data/json/itemgroups/item_groups.json

```diff
@@ -5305,6 +5307,7 @@
       [ "bleach", 20 ],
       [ "bandages", 50 ],
       [ "cotton_ball", 50 ],
+      [ "sponge", 20 ],
```

Comment:
Are sponges really part of dissection equipment?

Reply:
They're not used for the dissection itself, but for the post-dissection cleanup, as with the bleach and cotton balls.